iXBRLViewerPlugin/viewer/src/js/tableExport.js

```diff
@@ -159,17 +159,27 @@ TableExport.prototype._writeTable = function (data) {
 
             var cc = ws.getRow(i+1).getCell(j+1);
             if (cell.type == 'fact') {
-                cc.value = Number(cell.fact.value());
-                cc.numFmt = '#,##0';
-                ws.getColumn(j+1).width = 18;
-                /* Make this an option - apply presentation signs */
-                if (cell.negative) {
-                    cc.value = Math.abs(cc.value) * -1;
+                var numValue = Number(cell.fact.value());
+                if (Number.isNaN(numValue)) {
+                    cc.value = cell.fact.value();
+                    if (/<\/?[a-z][\s\S]*>/i.test(cc.value))  {
+                        cc.value = strInputCode.replace(/<\/?[^>]+(>|$)/g, "");
```

Comment:
What is the intent of this new `isNaN` branch, and how does it related to i18n?  These seem like they're trying to heuristically match `<` markup of some kind, but I don't know why.

Reply:
Looks like it's fixing #203.

It's unrelated to i18n so I've moved to a new branch https://github.com/paulwarren-wk/ixbrl-viewer/tree/203-non-numeric-table-export

I'll review it and raise a separate PR.